integration_test/functions/src/index.ts

```diff
@@ -107,9 +148,16 @@ function v1Tests(testId: string, accessToken: string) {
       .storage()
       .bucket()
       .upload('/tmp/' + testId + '.txt'),
-    testLab.startTestRun(firebaseConfig.projectId, testId),
+    testLab.startTestRun(firebaseConfig.projectId, testId, accessToken),
     // Invoke the schedule for our scheduled function to fire
-    callScheduleTrigger('v1-schedule', 'us-central1'),
+    callScheduleTrigger('v1-schedule', 'us-central1', accessToken),
+  ];
+}
+
+function v2Tests(testId: string, accessToken: string) {
```

Comment:
nit: please add return types for functions like this where it may be surprising.

Reply:
Fixed!